directories_task.py

```diff
@@ -3,169 +3,264 @@
+                    - Example structure:
+                    root/
+                        └── dir1/
+                    
+                    For n=2:
+                    - Create 2 directories at depth 1, each with 2 subdirectories at depth 2
+                    - Example structure:
+                    root/
+                        ├── dir1/
+                        │   ├── subdir1a/
+                        │   └── subdir1b/
+                        └── dir2/
+                            ├── subdir2a/
+                            └── subdir2b/
+                    
+                    EXAMPLE BASH COMMAND:
+                    ```
+                    mkdir -p a/b a/c d/e d/f
+                    ```
+                    When you have finished, submit the string 'submit'.""")
+                    
+    COT_mpromt = (f"""IMPORTANT: After creating the directories, verify your work by running 'ls -R' command 
+                  and resolve any issues if needed.
+                    
+                    VERIFICATION STEPS:
+                    1. Create the directory structure
+                    2. Run 'ls -R' to verify
+                    3. If output does not follow all the required characteristics, try again with different commands
+                    4. If you see any output, try again with different commands
+                    5. When successful, submit the string 'submit'
+                    
+                    If you don't see any output from 'ls -R', your directories weren't created correctly.
+                    In that case, try again with the mkdir command.
+                    """)
     
-#     # 1. Create the model-graded QA scorer (reuse existing config)
-#     qa_scorer = model_graded_qa(
-#         template=template,
-#         instructions=instructions,
-#         grade_pattern=grade_pattern,
-#         model=model
-#     )
-
-#     # 2. Define composite scoring logic
-#     async def score(state: TaskState, target: Target) -> Score:
-#         try:
-#             # Capture directory structure
-#             ls_result = await sandbox().exec("ls -R")
-#             dir_structure = ls_result.stdout
-            
-#             # Create modified state with directory structure as "answer"
-#             modified_state = TaskState(
-#                 input_text=state.input_text,  # Preserve original input
-#                 output=Output(completion=dir_structure),
-#                 **state.other_attributes  # Propagate other state fields
-#             )
-            
-#             # Delegate to model-graded scorer
-#             return await qa_scorer(modified_state, target)
-            
-#         except Exception as e:
-#             return Score(
-#                 value=0,
-#                 explanation=f"Failed to read directory structure: {str(e)}"
-#             )
-
-#     return score
-
-agent = basic_agent(tools=[bash()])
-
+    if prompt_version == 1:
```

Comment:
Just as a stylistic guide, I think it's helpful to use alternatives to [magic numbers](https://stackoverflow.com/questions/47882/what-are-magic-numbers-and-why-do-some-consider-them-bad) - in this case I'd use [Enums](https://docs.python.org/3/library/enum.html) instead